### DISCUSSION TITLE
LIFEPO4 Capacity Tester Ads1115 Question

### DISCUSSION THREAD

**Discussion by reset12** [arendst/Tasmota#17994] [Q&A]
Hello,
i have build a Capacity Tester for my new LIFEPO4  Cells.
I use a Wemos Mini, ADS1115 and a WCS1600 Hall Sensor for Current Measurement.

The Tester works, but i want to convert the Number Values from ADS1115 to Voltage and Current Values with Script Language.

>D
volt=0
>B
sensor12 S0
volt=Analog0
print %volt%

The Variablename Analog0 is not the Variable with the Value of  "ADS1115 Analog0   5653".

![Kapazitaetsmesser](https://user-images.githubusercontent.com/14790813/219959765-ef7c95c5-0e60-4ed3-a3cc-36ec3a891d17.jpg)
![Bildschirmfoto_2023-02-19_17-04-08](https://user-images.githubusercontent.com/14790813/219959810-343e2791-ce5d-4a4a-9142-eae42ecb29e5.png)

Thanks for your Help
Bruno

**Comment by barbudor** [comment]
Analog0 is the ESP internal ADC

You can get sensor values for the `>T` sections
Look at the examples to see how it works and compare with your SENSOR message

**Reply by barbudor** [reply]
And by the way, by using an ADS and a hall sensor you are making your life hard
An INA219 or INA226 would probably have been more suitable. Directly provides voltage and current without conversion.

**Reply by reset12** [reply]
The Current-Sensor i use can up to 100A .

Thank you for your Help.
